.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,9 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Support
```

Comment:
One could mention the Matrix Chat here, too?

Reply:
I honestly think that the Forum as first address for Support might be the better option, since it'll make problems way easier to find with your favourite search engine. The chat is awesome, but pushing people to the chat causes more repetitive questions which result in similar/same answers.

On the other hand 🤷🏻 it wouldn't hurt too much, I guess.